### ISSUE TITLE
[BUG]  K1C Camera Settings Control not possible to install

### ISSUE THREAD

**Issue by P-C-R** [Guilouz/Creality-K1-and-K1-Max#345]
### What happened?

Wanted to install. Get this error:

```
Type your choice and validate with Enter: 18

 ✗ Your camera version is not compatible!
```

### What did you expect to happen instead?

Install

### How to reproduce this issue?

Install on K1C with 1.3.3.5

**Comment by Stoony1** [comment]
same on my k1

**Comment by Guilouz** [comment]
It’s not an issue, that’s means you have the new model of webcam that is not compatible and can’t be adjusted.

**Comment by Hammerfest** [comment]
Yes, and everyone with a K1 variant (K1C/Max) should message Creality support requesting they return controls of the camera module to allow for better picture quality with added lighting.
